### ISSUE TITLE
Is there any porting documentation to the new API? 

### ISSUE THREAD

**Issue by darix** [Yubico/yubikey-manager#627]
Some distros might want to keep yubioath-desktop because packaging is easier compared to the flutter app.

I didnt find any documentation how to port things from the removed API to the new one.

**Comment by dainnilsson** [comment]
Changes and deprecations in the public API are generally documented in the NEWS file. However, yubioath-desktop uses some private APIs which are undocumented.

That said, I also want to strongly advise against packaging a much older version of the app as it lacks a lot of functionality, won't be compatible with newer devices, and may have unresolved bugs and security issues.

**Comment by darix** [comment]
well not even the yubikey-manager-qt is fully updated as you can e.g. see here https://github.com/Yubico/yubikey-manager-qt/issues/361

**Comment by dainnilsson** [comment]
That is correct. As stated in the README that project is no longer under active development. Yubico Authenticator 7 has replaced it and is far more capable.

**Comment by darix** [comment]
Which begs the question there are some PR open that would fix some of the issue to make it work with ykman > 5.0.1

will those ever be accepted or would it be better to just set the repository to archived?

**Comment by darix** [comment]
https://github.com/Yubico/yubioath-flutter

**Comment by dainnilsson** [comment]
@darix I'll bring the question to the team for discussion. I agree it's not great having things the way they are now. I don't really see much point in accepting the PRs with no intent to ever make a new release, so archiving seems like a good way to go.
